buttons/button_base.h

```diff
@@ -78,7 +88,12 @@ class ButtonBase : public Looper,
   const char* name_;
   enum BUTTON button_;
   uint32_t push_millis_;
+  bool short_click;
```

Comment:
global variables should end in _

Reply:
removed as it doesn't perform the check I thought it did.